aptos-move/aptos-vm-types/src/module_and_script_storage/state_view_adapter.rs

```diff
@@ -72,6 +82,62 @@ impl<'s, S: StateView, E: WithRuntimeEnvironment> AptosCodeStorageAdapter<'s, S,
         let storage = adapter.into_unsync_code_storage(runtime_environment);
         Self { storage }
     }
+
+    /// Drains cached verified modules from the code storage, transforming them into format used by
+    /// global caches. Should only be called when the code storage borrows [StateView].
+    pub fn into_verified_module_code_iter(
+        self,
+    ) -> Result<
+        impl Iterator<
```

Comment:
this description of item means the iterator is into iterator right? (we can get the items without cloning)?

Reply:
That's right